src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -364,8 +374,8 @@ private void appendNewLine(String content) {
   }
 
   public void clearContent() {
-    wait.forElementVisible(sourceModeTextArea);
-    sourceModeTextArea.clear();
+    wait.forElementVisible(textArea);
+    textArea.clear();
```

Comment:
I wonder why we clear() the element in one case and send backspaces in the other. It means the reason should be given in comments :)